team_management/views.py

```diff
@@ -31,3 +33,10 @@ def create(self, request, *args, **kwargs):
 			return Response(data=serialized_obj.data, status=status.HTTP_201_CREATED)
 			
 		return Response(data={"error": "Please make sure your [team name] is exist!"}, status=status.HTTP_400_BAD_REQUEST)
+
+	@action(detail=False,methods=["GET"])
```

Comment:
keknya harusnya detail=True,
trs tambahin self.object = self.get_objects()?
cc: @andrew-susanto wkwk

Reply:
owhh ini gua kepikirannya buat return current team aja sih fii

Reply:
oalah oke2 wkwk thank you.
Silakan merge @ferdifdi

Reply:
@andrew-susanto @luthfifahlevi bantu approve dulu wkwkwk